contrib/kube-local/README.md

```diff
@@ -0,0 +1,393 @@
+   - Matrix testing
+   - Continuous Integration (CI)
+   - Compare outputs from different components
+   - *[Add your use case here....]*
+
+## Demos
+
+### Demo 1 - Auto-install in quiet mode and execute kubectl commands
+Auto-installing kubernetes and it's requirements with CRI-O as runtime, conmon as runtime monitor and container network with network plugins.
+[![asciicast](https://asciinema.org/a/307812.png)](https://asciinema.org/a/307812)
+
+### Demo 2 - Continuous Integration use case
+An example for Continuous Integration use case. **Set kubernetes and CRI-O to version 1.13** and execute a script to validate the environment. After execution, remove every file/dir generated by kube-local using **-r option**.
+[![asciicast](https://asciinema.org/a/307815.png)](https://asciinema.org/a/307815)
+
+### Demo 3 - Manual Install
+Installing a devel environment manually.
+[![asciicast](https://asciinema.org/a/307825.png)](https://asciinema.org/a/307825)
+
+## Supported distros
+  - [Fedora Linux](https://getfedora.org/)
+  - [Red Hat Enterprise Linux](https://redhat.com) *(needs help/testers)*
+  - [openSUSE](https://www.opensuse.org/) *(needs help/testers)*
+  - [Debian GNU/Linux](https://www.debian.org/) *(needs help/testers)*
+  - [Ubuntu](https://www.ubuntu.com)  (needs help/testers)
+
+## Supported runtime
+  - [CRI-O](https://github.com/cri-o/cri-o) - Open Container Initiative-based implementation of Kubernetes Container Runtime Interface
+
+## Supported container orchestration
+  - [Kubernetes](https://github.com/kubernetes/kubernetes) - Production-Grade Container Scheduling and Management
+
+## How it works?
+   kube-local is based on **bash** as most of the code in kubernetes/hack. There is a configuration file called **kube-local.profile** which specify all the configurations.
+
+```sh
+$ ./kube-local -h
+Usage: ./kube-local [OPTION]...
+
+./kube-local - A script for helping deploy a local  cluster
+
+ -a, --autoinstall                      Autoinstall mode, don't ask questions during deploy.
+                                        See profile file for additional info and options regarding autoinstall
+
+ -p, --profile=FILENAME                 A profile to be loaded and used during execution
+ -c, --command=[FILENAME]               Execute a command(s) in the cluster (requires -o)
+ -h, --help                             Display this message and exit
+ -o, --output                           Output (requires -c)
+ -r, --run-service-once                 Run the command and stop kubernetes service. Cleanup env after command execution. (requires -c and -o)
+ -q, --quiet                            Quiet mode. Do not print any message into stdout. Messages can be read from log file.
+ -u, --uninstall                        Uninstall all files/dirs generated from this script
+ -s, --container-orchestration-version  Specify the container ORCHESTRATION version
+ -k, --container-runtime-version        Specify the container RUNTIME version
+```
+
```

Comment:
Some of the sentences for the above options end with a period ".", others do not.  I prefer that they all ended with a period, regardless, they should be consistent.